enhancements/windows-containers/health-management.md

```diff
@@ -225,6 +228,26 @@ validate that essential files have not been tampered with.
 }
 ```
 
+The `environmentVars` key defines the desired configurations for environment variables on Windows instances.
+Whenever there is a modification to these configurations, the WICD controller is triggered to reconcile the changes and
+ensure that the updated values are propagated to the Windows instances.
+```json
+ {
+  "name": "name of the environment variable",
+  "value": "value the variable should be set to on the instance"
+}
+```
+
+The `watchedEnvironmentVars` key comprises a list of specific environment variables monitored by WICD.
+Should any of these watched environment variables be absent from the current environment variable configuration,
+WICD proactively removes them from the instance, thereby ensuring the environment variables adhere to the configmap
+specifications.
+```json
+{
+  "name": "name of the environment variable"
+}
+```
```

Comment:
Did you consider a property in  `environmentVars` , i.e. `watched: true`  to keep track of the environment variables that need monitoring instead of proposing a separated list (`watchedEnvironmentVars`)?

Reply:
@jrvaldes this is an already implemented schema now. 
The reason for keeping it a separate field is to enable WMCO to maintain a fixed env variable definition that WICD can use to compare and correct the env variables.  EnvironmentVars key contents will change as the proxy resource changes but watcheEnvironmentVars list will remain constant.

Reply:
I follow. Thanks for the explanation.